cabot/cabotapp/models.py

```diff
@@ -141,6 +141,11 @@ class Meta:
         help_text='Gist, Hackpad or Refheap js embed with recovery instructions e.g. '
                   'https://you.hackpad.com/some_document.js'
     )
+    hipchat_room_id = models.IntegerField(
```

Comment:
Can we also take a hitchat server url and API key as parameters?

Reason: We may be able to point to a global hipchat server in the future. We want to alert merchants on their hipchat server room etc.

Alternately, can we create a db model for hipchat configuration?

The plugin should receive, hipchat_url, api_key and room_id as parameters - with sane defaults...